src/Microsoft.AspNetCore.Mvc.Core/project.json

```diff
@@ -40,6 +40,7 @@
       "type": "build",
       "version": "1.1.0-*"
     },
+    "Microsoft.Extensions.Logging": "1.1.0-preview2-22526",
```

Comment:
How should I handle scenarios where ILoggerFactory is not initialized and is null when BodyModelBinder ctor is called? I want to call CreateLogger in the constructor to initialize an ILogger. Is adding null tests for the logger before any logging a good idea? It seemed to me that will make the code look untidy.